README.md

```diff
@@ -3932,7 +3932,7 @@ Other Style Guides
 **Podcasts**
 
   - [JavaScript Air](https://javascriptair.com/)
-  - [JavaScript Jabber](https://devchat.tv/js-jabber/)
+  - [JavaScript Jabber](https://topenddevs.com/podcasts/javascript-jabber)
```

Comment:
https://devchat.tv/js-jabber/ is redirecting to https://topenddevs.com/podcasts/javascript-jabber link only , do we still need this change ?

Reply:
it's fine to update to include the redirect, but it's also find to just let the redirect continue, since in theory it'll never break.

Reply:
It will not break , but what if 'https://devchat.tv/js-jabber/' redirects to a different link in future ?